Added debugging functionality.

GNikit opened the pull request:
Hi,
Not having the option to add breaking points for debugging has been bothering me for quite some time and I think it is unreasonable to have to install a completely new extension about this, so I took the liberty of adding them in the extension.
The implemented changes are trivial overall.

GNikit commented:
I guess this branch should be merged to master after the dependabot PRs.

krvajal commented:
Hi @GNikit and thanks for contributing.
The PR looks good. Maybe add some info in the docs (Readme.md) about how to use this feature and also which OS does it support?

GNikit commented:
> Hi @GNikit and thanks for contributing.
> The PR looks good. Maybe add some info in the docs (Readme.md) about how to use this feature and also which OS does it support?

So I think this is done, I updated the README, I also updated the way we define the debugger, there is no need to specify the type of the debugger `cppdb` in `package.json`. My first attempt was actually using a deprecated way of doing it. 

However, I feel that I should mention this to make things clear. 
The only thing that I have added is the ability to set breaking points in Fortran files, nothing else.
By adding the dependency on Microsoft's C/C++ `ms-vscode.cpptools` we can ensure that the users will have the underlying infrastructure to launch C++-like programs, Fortran falls under this category, through the Debug console of VS Code. **So there is no explicit dependence of the extension on `ms-vscode.cpptools`**.

I personally don't think that is an issue, but given how this is your extension I feel that I should let you know.
